### DISCUSSION TITLE
Possibility to Upload docs with size above 70mb

### DISCUSSION THREAD

**Discussion by AlexD-Bearing** [openkfw/TruBudget#853] [General]
At the workflow item level, giving the possibility to upload documents which size are above 70mb

**Comment by laurenzhonauer** [comment]
Idea:

Use [connect-busboy](https://github.com/mscdex/connect-busboy).
It is an express middleware for uploading files via busboy.
Could be used [right here](https://github.com/openkfw/TruBudget/blob/master/storage-service/src/index.ts#L93)

For Reference read [this article](https://medium.com/@vecera.petr/how-to-handle-large-file-upload-with-nodejs-express-server-7de9ab3f7af1)
